docs/developers/infrastructure.md

```diff
@@ -0,0 +1,42 @@
+# Infrastructure Setup
+The whole infrastructure of data package registry is built on AWS services.
+## Domain and certificate setup:
+AWS has domain hosting service called ```Route53```. This is the only place we are not using aws service.
+
+The extrnal domain: **datapackaged.com**
+
+We mainly have 2 parts:
+- ApiGateWay [Custom domain]
+- S3
+    - Static Files [static.datapackages.com]
+    - Data File [bits.datapackaged.com]
+
+### Api Gateway:
+For api gateway having custom domain we have to add generated
```

Comment:
Think "having" is extra here.

> For api gateway custom domain we have to add generated certificates:

Also suggest adding "certificates:" at the end